pkg/storage/ingest/fetcher.go

```diff
@@ -24,6 +25,11 @@ import (
 	"github.com/grafana/mimir/pkg/util/spanlogger"
 )
 
+const (
+	unknownBrokerFranzGoErrorString    = "unknown broker"
+	chosenBrokerDiedFranzGoErrorString = "the internal broker struct chosen to issue this request has died--either the broker id is migrating or no longer exists"
```

Comment:
```suggestion
	errUnknownBroker    = "unknown broker"
	errBrokerDied = "the internal broker struct chosen to issue this request has died--either the broker id is migrating or no longer exists"
```

Reply:
nit: a bit more concise, I don't think the long name add anything if we're documtning their usage in the `case` loop. The prefix is just for consistency and easily grep in the file.